### PR TITLE
Replace setPadding with updatePadding

### PR DESCRIPTION
**Pull request by NotWoods**
This makes code easier to read when its just setting padding on one side.

### CODE REVIEW

**Comment by person808** [comment]
core-ktx has updatePadding and updatePaddingRelative. https://developer.android.com/reference/kotlin/androidx/core/view/package-summary#(android.view.View).updatePadding(kotlin.Int,%20kotlin.Int,%20kotlin.Int,%20kotlin.Int)

**Comment by NotWoods** [comment]
I didn't know about updatePadding, that makes more sense to use here. The 0 values are really just used as defaults.

File: buildSrc/src/main/java/Dependencies.kt [hunk 1]
```diff
@@ -15,7 +15,7 @@ object Versions {
     const val androidx_appcompat = "1.2.0-rc01"
     const val androidx_biometric = "1.1.0-alpha01"
     const val androidx_coordinator_layout = "1.1.0-rc01"
-    const val androidx_constraint_layout = "2.0.0-beta6"
```
Comment (eliserichards) [inline buildSrc/src/main/java/Dependencies.kt:18]: I'm guessing there was a reason that we were using the beta version here. Can you look for the blame of that and double check that it's ok to downgrade to the stable version?

File: settings.gradle [hunk 2]
```diff
@@ -64,7 +64,7 @@ if (localProperties != null) {
         // As above, hacks to execute .py files on Windows.
         def publishAcCmd = [];
         if (System.properties['os.name'].toLowerCase().contains('windows')) {
-            publishAcCmd << "py";
+            publishAcCmd << "python3";
```
Comment (eliserichards) [inline settings.gradle:67]: What's this for? Can we move to a separate PR if it's not relevant to the padding changes?

File: buildSrc/src/main/java/Dependencies.kt (hunk 1 above)
Comment (NotWoods) [reply buildSrc/src/main/java/Dependencies.kt:18]: Oops, I didn't mean to put this change in this PR

File: settings.gradle (hunk 2 above)
Comment (NotWoods) [reply settings.gradle:67]: Same here 😅
